src/main/java/org/opentripplanner/routing/api/request/preference/StreetPreferences.java

```diff
@@ -179,6 +179,13 @@ public Builder withIntersectionTraversalModel(IntersectionTraversalModel model)
       return this;
     }
 
+    public Builder withMaxAccessEgressDuration(
+      DurationForEnum<StreetMode> maxAccessEgressDuration
+    ) {
+      this.maxAccessEgressDuration = maxAccessEgressDuration;
```

Comment:
I think we need to merge this with `this.maxAccessEgressDuration`, not just overwrite